tests/utils/helper.py

```diff
@@ -38,3 +50,40 @@ def __init__(self, filename):
 
     def __getattr__(self, key):
         return getattr(self._inventory, key)
+
+
+def _find_resource_name(resource):
+    for resource_name, api in NAMESPACED_RESOURCES.items():
+        if resource == resource_name or resource == resource_name.strip('_'):
```

Comment:
For my own curiosity, why the strip(_')

Reply:
to allow 'daemonset' and 'daemon_set' (and eventually 'DaemonSet' if someone ask for)

Reply:
Pure cosmetic for feature file. The "daemon_set" version is not what we are used for (because of kubectl that strip underscore)

Reply:
I'm confused. `str.strip` only strips at the front and end of a string, so how would this work?

Reply:
hmm indeed a bug. It's not yet used in this PR (this on use `persistent_volume` which looks like better than `persistentvolume` (will be in the following).
```
resource.lower() = resource_name.replace('_', '')
```
Remove the ability for `daemon_set` (but introduce  `DaemonSet`). Does it work for both of you

Reply:
yes